### DISCUSSION TITLE
compañeros alguno de ustedes podria ayudarme :(?

### DISCUSSION THREAD

**Discussion by EinarDvls** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#363] [Q&A]
hola a todos antes que nada, me gustaría saber si alguno de ustedes podría ayudarme con mi practica de CSS ya que lo único que me falta es alinear la imagen que va al costado del contacto de la misma, ya intente de todo y no puedo alinearla, tendrán alguna idea? 
![image](https://user-images.githubusercontent.com/99060661/158027358-557ab0c7-d5fe-461f-b3e1-4c18971870b9.png)
![image](https://user-images.githubusercontent.com/99060661/158027397-85a4656e-6eb6-45b0-a25a-5dcbb127acc7.png)

![image](https://user-images.githubusercontent.com/99060661/158027417-948ba4f3-1423-4fef-acf5-e7afac283a49.png)

**Comment by doguedogue** [comment]
Hay varias formas
- métela dentro de un div y dale text-align (pero es imagen, si, así es)
-  flexbox 
-  etc.

Vas por buen camino, mira las clases u otros videos de css, tu puedes!

**Reply by EinarDvls** [reply]
no se arregla de ninguna forma, la verdad no se que estoy escribiendo mal

**Comment by Ingrzs** [comment]
Mira te recomiendo esto
Al contenedor padre dale un width del 100% y dale la propiedas display flex
que el contenedor tenga dos hijo
contenedor 1(lo del texto) width 50%
contenedor 2 (imagen) witdh 50%

Con eso ya tendras la imagen al lado tal como esta en la practica

**Comment by FernandoBarajasValdivia** [comment]
O con grids de 50% y 50%

**Comment by hackit92** [comment]
Yo te recomendaría crear una clase para el div principal de la sección que defina un grid-template de 50% 50% y de esa forma los divs internos se acomodarán en esas 2 columnas.
